source/renderer/app/components/status/NetworkStatus.js

```diff
@@ -190,7 +407,28 @@ export default class NetworkStatus extends Component<Props, State> {
       isStaging,
       isMainnet,
     } = this.props;
-    const { data, isNodeRestarting } = this.state;
+
+    const {
+      platform,
+      platformVersion,
+      cpu,
+      ram,
+      availableDiskSpace,
+    } = systemInfo;
+
+    const {
+      daedalusVersion,
+      daedalusProcessID,
+      daedalusMainProcessID,
+      isInSafeMode,
+      cardanoVersion,
+      cardanoProcessID,
+      cardanoAPIPort,
+      cardanoNetwork,
+      daedalusStateDirectory,
+    } = coreInfo;
+
+    const { isNodeRestarting } = this.state;
     const isNTPServiceReachable = !!localTimeDifference;
```

Comment:
@DeeJayElly we need a better check here (I know this is not related to your changes!) - currently if time diff is zero we show `NTP service unreachable` and we should show `0` value!

Reply:
@nikolaglumac done

Reply:
@DeeJayElly I am not happy with this :(
> const isNTPServiceReachable =  localTimeDifference > 0 ? !!localTimeDifference : '0';
`isNTPServiceReachable` should be a boolean and in your case it is either a boolean or zero.
What we really want here is for `isNTPServiceReachable` to be true in case `localTimeDifference` is zero (which wasn't the case before).

Perhaps the solution is to use the following code:
```
const isNTPServiceReachable =  localTimeDifference != null;
```

Please try that one and test the outcomes 🙏